src/redux/saga/employer.ts

```diff
@@ -1,19 +1,55 @@
-import { put, takeLatest } from 'redux-saga/effects'
+import { put, takeLatest, select } from 'redux-saga/effects'
 import axios from 'axios'
 
-import { registerEmployerSuccess } from '../../redux/actions/employer'
+import { AppState } from '../types'
 
-function* registerEmployerSaga(credentials: Credential) {
+import {
+  registerEmployerSuccess,
+  registerEmployerFail,
+  loginEmployerSuccess,
+  loginEmployerFail,
+} from '../../redux/actions/employer'
+
+const credential = (state: AppState) => state.employer.credential
+function* registerEmployerSaga() {
   try {
-    const req = yield axios.post('/employer', { credentials })
+    const credentialData = yield select(credential)
+    const req = yield axios.post('/employer', {
+      info: {},
```

Comment:
I believe you can remove this also

Reply:
I don't need to make a post request when registering? Or did you mean that I can remove the info?

Reply:
Yes I meant to remove `info`

Reply:
Ok. Is the skills removed from credentials already? The jobseeker register request is still not working. It doesn't give an error but just keeps loading.